src/components/SelectionList/Search/ReportListItem.tsx

```diff
@@ -77,6 +77,9 @@ function ReportListItem<TItem extends ListItem>({
         styles.overflowHidden,
         item.isSelected && styles.activeComponentBG,
         isFocused && styles.sidebarLinkActive,
+        // Removing some of the styles because they are added to the parent OpacityView via animatedHighlightStyle
+        {backgroundColor: 'unset'},
```

Comment:
This style overriding caused the following issue:
- https://github.com/Expensify/App/issues/50046

Which we fixed by moving the style up, under the `overflowHidden` and changing it to `styles.bgTransparent` for compatibility with native.